### PR TITLE
add Imd writer option

### PR DESCRIPTION
**Pull request by wybren1971**
Changes in this pull request:
Add the ability to write imd images to file.
Update the documentation regarding the imd image possibilities.
make the HPLif format read and write
set the ibm format as default format in the gui version

### CODE REVIEW

File: lib/imagewriter/imdimagewriter.cc [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+					numSectorsinTrack = 0;
+				nanoseconds_t RATE = 0;
+				if (sector->clock > 0)
+				{
+					RATE = 1000000.0 / sector->clock;
+				} else
+				{
+					switch (dataRate)
+					{
+					case ImdOutputProto::RATE_HD : RATE = 1000;
```
Comment (davidgiven) [inline lib/imagewriter/imdimagewriter.cc:225]: This probably want `break` statements!

**Comment by davidgiven** [comment]
Thank you very much! LGTM, but I did spot one issue and left a comment...

**Comment by wybren1971** [comment]
i added the breaks to the switch statement. spotted also one in the reader which i updated also

File: lib/imagewriter/imdimagewriter.cc (hunk 1 above)
Comment (wybren1971) [reply lib/imagewriter/imdimagewriter.cc]: updates the code with break statements...

**Pull request merged**